wiki/Tournaments/YSC/2017/en.md

```diff
@@ -8,7 +8,7 @@ tags:
 
 ![YSC 2017 logo](img/logo.png)
 
-The **Yaz's Summer Cup 2017** (***YSC 2017***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) and sponsored by Ferocity Esports. It was the first instalment of Yaz's Summer Cup.
+The **Yaz's Summer Cup 2017** (***YSC 2017***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) and sponsored by Ferocity Esports. It was the first instalment of the Yaz's Summer Cup.
```

Comment:
The tournament template asks for `the` to be included here but to me, it doesn't read nice. Would like your opinion @Walavouchey

Reply:
You're pretty much right. Not all tournament titles can really take a "the". For example, `the Corgi Cup` and `the osu! World Cup` work, while `the Corgi Cup V` or `the osu! World Cup 2020` do not because of the number. `thing #20` works but `the thing #20` implies that `the thing` is a thing of its own (i.e. "The Thing"). This example here also doesn't work for similar reasons `the yaz's thing` would imply that "the yaz's thing" is a thing of its own (and also ungrammatical). I suppose this should be changed and clarified in the template.

Reply:
I agree. I already spotted the points raised in #5747, so not worth continuing the discussion much here.

As a resolution for this PR, I am going to remove the `the` from `{TITLE}` but remain at {SERIES_TITLE}`. I think that reads more clearly.

If there is a resolution to this at any point, it can be changed then.

Reply:
if you are going to remove the `the`s, then remove in both `{TITLE}` and `{SERIES_TITLE}` as i explained above